### DISCUSSION TITLE
CIS VPC: How do I maximize subnet size when not using all availability zones?

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#790] [How do I]
I am using the CIS VPC module, and I do not need a private-persistence tier group of subnets, and I only want to use 3 AZs in us-east-1. 

I would like only two tiers of subnets spread across the same 3 AZs, and I want to exhaust my VPC's CIDR range across those 6 subnets. My starting CIDR is `172.20/22`. 

---

<ins datetime="2023-12-11T17:20:38Z">
  <p><a href="https://support.gruntwork.io/hc/requests/110637">Tracked in ticket #110637</a></p>
</ins>

**Comment by gitsstewart** [comment]
@zachreborn are you able to answer this question or point us in the right direction?
